packages/eslint-config-eslint/tsconfig.json

```
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "allowSyntheticDefaultImports": false,
```

```suggestion
        "allowSyntheticDefaultImports": true,
```

I think the idea was to set `allowSyntheticDefaultImports` to `true` to enforce the syntax `import eslintConfigESLint from "eslint-config-eslint";` over `import * as eslintConfigESLint from "eslint-config-eslint";` in our tests.

`allowSyntheticDefaultImports` makes TypeScript behave more similarly to Node.js when it imports a CommonJS file, and that's what the team is more familiar with.